ci/gpu/build.sh

```diff
@@ -42,11 +43,13 @@ nvidia-smi
 
 logger "Activate conda env..."
 source activate gdf
-conda install "rmm=$MINOR_VERSION.*" "nvstrings=$MINOR_VERSION.*" "cudatoolkit=$CUDA_REL" \
-              "dask>=2.1.0" "distributed>=2.1.0" "numpy>=1.16" "double-conversion" \
-              "rapidjson" "flatbuffers" "boost-cpp" "fsspec>=0.3.3" "dlpack" \
-              "feather-format" "cupy>=6.2.0" "arrow-cpp=0.14.1" "pyarrow=0.14.1" \
-              "fastavro>=0.22.0" "pandas>=0.24.2,<0.25" "hypothesis"
+conda install "cudf>=0.9" "dask-cudf>=0.9" "cudatoolkit=$CUDA_REL" \
+              "dask>=2.3.0" "distributed>=2.3.2" "numpy>=1.16" "cupy>=6.2.0"
```

Comment:
I don't think `cupy>=6.2.0` is available on any of the conda channels we've been using. I would recommend using the one from @jakirkham's channel instead, I've been using it and it's working great:

```
conda install nvidia::nccl=2 jakirkham/label/cupy::cupy>=6.2.0
```

Reply:
> Ah it was been copied to rapidsai. So should be there.

Thanks for clarifying @jakirkham. We can ignore my comment then.